src/rest/connector/libs/apic/implementation.py

```diff
@@ -269,17 +269,17 @@ def get(self, dn, query_target='self', rsp_subtree='no', \
 
     @BaseConnection.locked
     @isconnected
-    def post(self, dn, payload, expected_status_code=requests.codes.ok,
-             timeout=30):
+    def post(self, dn, payload, xml_payload=False,
+             expected_status_code=requests.codes.ok, timeout=30):
         '''POST REST Command to configure information from the device
 
         Arguments
         ---------
 
             dn (string): Unique distinguished name that describes the object
                          and its place in the tree.
-            payload (dict): Dictionary containing the information to send via
-                            the post
+            payload (dict): Information to send via the post command
+            xml_payload (bool): Set to True if payload is in XML format
```

Comment:
Same idea here, Please update `payload (dict):` if the data type isn't always a dict